src/App.tsx

```diff
@@ -10,33 +12,23 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectTab] = useState<Tab>(tabs[0]);
+
+  const onTabSelected = (tab: Tab) => {
+    setSelectTab(tab);
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        {`Selected tab is ${tabs.find(tab => (selectedTab.id === tab.id))?.title}`}
```

Comment:
to me it looks you already have the selected tab and I don't see any reason to look for it again)